conf/evolutions/default/242.sql

```diff
@@ -0,0 +1,35 @@
+# --- !Ups
+DELETE FROM label_validation
```

Comment:
Should also add a comment saying something like:
> Delete duplicate validations, keeping only the most recent one.

And I'd include the comments you added below here instead, feels more appropriate here rather than next to the part where you add the constraint.